v2/pkg/core/inputs/hybrid/hmap.go

```diff
@@ -112,12 +120,12 @@ func (i *Input) scanInputFromReader(reader io.Reader) {
 			i.expandASNInputValue(scanner.Text())
 			continue
 		}
-		i.normalizeStoreInputValue(scanner.Text())
+		i.NormalizeStoreInputValue(scanner.Text())
 	}
 }
 
-// normalizeStoreInputValue normalizes and stores passed input values
-func (i *Input) normalizeStoreInputValue(value string) {
+// NormalizeStoreInputValue normalizes and stores passed input values
+func (i *Input) NormalizeStoreInputValue(value string) {
```

Comment:
This method should be used tied with uncover as it's part of a interface and there can be other input providers. Instead the logic should be made common and utilized by both packages probably. Not sure how trivial that'll be

Reply:
Added Set() method to input provider and renamed this method to Set()